Caboodle/Battery/Battery.uwp.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Battery
+    {
+        static Windows.Devices.Power.Battery DefaultBattery => 
+            Windows.Devices.Power.Battery.AggregateBattery;
+
+        public static double ChargeLevel
+        {
+            get
+            {
+                var finalReport = DefaultBattery.GetReport();
+                var finalPercent = -1.0;
+
+                if (finalReport.RemainingCapacityInMilliwattHours.HasValue && finalReport.FullChargeCapacityInMilliwattHours.HasValue)
+                {
+                    finalPercent = (double)finalReport.RemainingCapacityInMilliwattHours.Value /
+                                     (double)finalReport.FullChargeCapacityInMilliwattHours.Value;
+                }
+                return finalPercent;
+            }
+        }
+        public static BatteryState State
+        {
+            get
+            {
+                var percentage = ChargeLevel;
+
+                if (percentage >= 1.0)
+                    return BatteryState.Full;
+                if (percentage < 0)
+                    return BatteryState.Unknown;
```

Comment:
I think this should go after the state checks. Could it be the case that it is at 0, and charging.

Reply:
Sounds good